flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/RuntimeEnvironment.java

```diff
@@ -316,6 +319,11 @@ public TaskOperatorEventGateway getOperatorCoordinatorEventGateway() {
         return operatorEventGateway;
     }
 
+    @Override
+    public ThroughputCalculator getThroughputMeter() {
+        return throughputCalculator;
+    }
+
```

Comment:
nit: unrelated change

Reply:
yes, I moved all "final" attributes all together along with their get method for easy reading. 

I can maybe create a hotfix on this if you inisit.

Reply:
I see, I think it's fine to use the same commit.